Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -468,29 +471,30 @@ private void SendControllerStateEvents(float time)
             {
                 if (CurrentButtonStates.IsMenuButtonDown)
                 {
-                    InputManager.Instance.RaiseSourceDown(this, controllerId, InteractionSourcePressType.Menu);
+                    InputManager.Instance.RaiseSourceDown(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Menu);
                 }
                 else
                 {
-                    InputManager.Instance.RaiseSourceUp(this, controllerId, InteractionSourcePressType.Menu);
+                    InputManager.Instance.RaiseSourceUp(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Menu);
                 }
             }
 
             if (CurrentButtonStates.GraspStateChanged)
             {
                 if (CurrentButtonStates.IsGrasped)
                 {
-                    InputManager.Instance.RaiseSourceDown(this, controllerId, InteractionSourcePressType.Grasp);
+                    InputManager.Instance.RaiseSourceDown(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Grasp);
                 }
                 else
                 {
-                    InputManager.Instance.RaiseSourceUp(this, controllerId, InteractionSourcePressType.Grasp);
+                    InputManager.Instance.RaiseSourceUp(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Grasp);
                 }
             }
+#endif
         }
 
         /// <summary>
-        /// Sends the events for hand visibility changes & controller connect/disocnnect.
+        /// Sends the events for hand visibility changes & controller connect/disconnect.
```

Comment:
nit: this ampersand is highlighted red for me on GitHub. I guess change it to an "and" to make it happy?

![image](https://user-images.githubusercontent.com/3580640/30893345-7ca21ae8-a2f2-11e7-92f3-62d4cbcc85fe.png)

Reply:
I see it too, but I don't think it's anything to worry about.  Just some formatting thing.

Reply:
@keveleigh I figured out why it's marked red.  The `&` should be escaped to `&amp;`